app/src/ui/delete-branch/delete-branch-dialog.tsx

```diff
@@ -45,9 +45,11 @@ export class DeleteBranch extends React.Component<
         onDismissed={this.props.onDismissed}
         disabled={this.state.isDeleting}
         loading={this.state.isDeleting}
+        role="alertdialog"
+        ariaDescribedBy="delete-branch-confirmation-message delete-branch-confirmation-message-remote"
```

Comment:
I hope screen readers / accessibility tools are resilient enough if one of these ids doesn't exist in the DOM, otherwise we should only add `delete-branch-confirmation-message-remote` when the element with that `id` is added to the DOM.

Reply:
I have tried a scenario where the id doesn't exist and it seems to safely ignore without consequence.